src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.Unix.cs

```diff
@@ -79,6 +79,11 @@ internal bool BackendSupportsCustomCertificateHandling
         public void HttpClientUsesSslCertEnvironmentVariables(bool setSslCertDir, bool createSslCertDir,
             bool setSslCertFile, bool createSslCertFile, bool expectedFailure)
         {
+            if (expectedFailure && RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Comment:
As #28002 indicates, we need to fix this test `using (HttpClient client = CreateHttpClient(useSocketsHttpHandler))` instead of `new HttpClient()`.

Since now SocketsHttpHandler is the default (when do`new HttpClient()`), this test is guaranteed to fail. So we don't add `UseSocketsHttpHandler` check here.